sktime/utils/plotting.py

```diff
@@ -114,6 +123,24 @@ def plot_series(
 
         plot_func(x=x, y=y, ax=ax, marker=marker, label=label, color=color)
 
+    # plot prediction intervals if present
+    if pred_int is not None:
+        check_pred_int(pred_int)
+        # check same conditions as for earlier indices
+        if all([x in index for x in pred_int.index]):
+            ax.fill_between(
+                ax.get_lines()[-1].get_xdata(),
+                pred_int.lower,
+                pred_int.upper,
+                alpha=0.3,
+                color=ax.get_lines()[-1].get_c(),
+                label="prediction intervals",
+            )
+        else:
+            raise ValueError("pred_int and x axis indices does not match")
+    else:
```

Comment:
@Ifeanyi30 and @mloning do we actually want to raise an error when `pred_int` is None? There are probably many use cases where we want to plot a series but not its prediction interval. I'd think we want to plot the interval if a value is passed and otherwise not do anything (e.g. remove the else that happens if `pred_int is not None` is `False`.